hooks/pre-commit

```diff
@@ -0,0 +1,52 @@
+#!/bin/sh
+#
+# An example hook script to verify what is about to be committed.
+# Called by "git commit" with no arguments.  The hook should
```

Comment:
Does this mean that this will not run when running `git commit -m 'my commit'` for instance?

Reply:
I guess. Let's test